pkg/cwversion/component/component.go

```diff
@@ -7,20 +7,21 @@ package component
 
 // Built is a map of all the known components, and whether they are built-in or not.
 // This is populated as soon as possible by the respective init() functions
-var Built = map[string]bool {
-	"datasource_appsec":     false,
-	"datasource_cloudwatch": false,
-	"datasource_docker":     false,
-	"datasource_file":       false,
-	"datasource_journalctl": false,
-	"datasource_k8s-audit":  false,
-	"datasource_kafka":      false,
-	"datasource_kinesis":    false,
-	"datasource_loki":       false,
-	"datasource_s3":         false,
-	"datasource_syslog":     false,
-	"datasource_wineventlog":false,
-	"cscli_setup":           false,
+var Built = map[string]bool{
+	"datasource_appsec":       false,
+	"datasource_cloudwatch":   false,
+	"datasource_docker":       false,
+	"datasource_file":         false,
+	"datasource_journalctl":   false,
+	"datasource_k8s-audit":    false,
+	"datasource_kafka":        false,
+	"datasource_kinesis":      false,
+	"datasource_loki":         false,
+	"datasource_s3":           false,
+	"datasource_syslog":       false,
+	"datasource_wineventlog":  false,
+	"datasource_victorialogs": false,
```

Comment:
this should be listed in the Makefile too -- the only reason to "register" datasources is to optionally exclude from the builds with a tag, and still list them in "cscsli version"

Reply:
Added this to Makefile in https://github.com/crowdsecurity/crowdsec/pull/3310/commits/f4603e1b45bbad65d07cb03b288637f5093db92c, thanks for noticing